src/OrchardCore.Modules/OrchardCore.HealthChecks/Startup.cs

```diff
@@ -21,7 +21,7 @@ public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder ro
         {
             var healthCheckOptions = serviceProvider.GetService<IOptions<HealthChecksOptions>>().Value;
 
-            app.UseHealthChecks(healthCheckOptions.Url);
+            routes.MapHealthChecks(healthCheckOptions.Url);
```

Comment:
It does not do exactly the same, `MapHealthChecks()` will build a new app builder (we have one here) to create a new pipeline (will shortcut the tenant pipeline), so I think that still using `UseHealthChecks()` is better.

Reply:
[If you look at the source you will see that `UseHealthChecks` uses `MapWhen()` where `MapHealthChecks` uses `Map()`](https://stackoverflow.com/questions/72384646/usehealthchecks-vs-maphealthchecks)

Reply:
Also in ASP.NET docs, they are using `MapHealthChecks` instead https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-7.0#basic-health-probe

Reply:
These are not necessarily good reasons, better to look at what they are doing and to know in which context we are.